homeassistant/components/microbees/button.py

```diff
@@ -0,0 +1,53 @@
+"""Button integration microBees."""
+from typing import Any
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesActuatorEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the microBees button platform."""
+    coordinator: MicroBeesUpdateCoordinator = hass.data[DOMAIN][
+        entry.entry_id
+    ].coordinator
+    async_add_entities(
+        [
+            MBButton(coordinator, bee_id, button.id)
+            for bee_id, bee in coordinator.data.bees.items()
+            if bee.productID == 51
+            for button in bee.actuators
+        ]
+    )
```

Comment:
```suggestion
    async_add_entities(
        MBButton(coordinator, bee_id, button.id)
        for bee_id, bee in coordinator.data.bees.items()
        if bee.productID == 51
        for button in bee.actuators
    )
```

You can avoid creating a list here